Userland/Libraries/LibGUI/DatePickerDialog.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023, David Ganz <david.g.ganz@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/String.h>
+#include <LibCore/DateTime.h>
+#include <LibGUI/Calendar.h>
+#include <LibGUI/Dialog.h>
+#include <LibGUI/Model.h>
+
+namespace GUI {
+
+class DatePickerDialog : public Dialog {
```

Comment:
Not a comment on this line actually, but please add `DatePickerDialog` to `LibGUI/Forward.h`.